### DISCUSSION TITLE
Coloring Issue with Maintained Sceneform Fork

### DISCUSSION THREAD

**Discussion by davidAnPrintell** [google/filament#5136] [Q&A] [answered]
Hey,

Recently we come across some coloring issues on a maintained Sceneform Fork (https://github.com/SceneView/sceneform-android), which obviously based on filament.

We've been wondering if there is some change in Filament, which affects the color grading of sceneforms camera view.

Issue i'm talking about: https://github.com/SceneView/sceneform-android/issues/295
Related issue: https://github.com/SceneView/sceneform-android/issues/252

Appreciate any help or discussions!

**Comment by romainguy** [comment, marked as answer]
See https://github.com/google/filament/issues/3909. The workaround is to use the FILMIC tone mapping operator. It'll fix your problem.
